core/src/main/java/tc/oc/pgm/listeners/AntiGriefListener.java

```diff
@@ -60,10 +59,11 @@ public void rightClickDefuse(final PlayerInteractEntityEvent event) {
     this.participantDefuse(event.getPlayer(), event.getRightClicked());
   }
 
-  @EventHandler(priority = EventPriority.HIGHEST, ignoreCancelled = true)
-  public void leftClickDefuse(final PlayerAttackEntityEvent event) {
-    this.participantDefuse(event.getPlayer(), event.getLeftClicked());
-  }
+  // TODO: This is only in SportPaper
+  //  @EventHandler(priority = EventPriority.HIGHEST, ignoreCancelled = true)
+  //  public void leftClickDefuse(final PlayerAttackEntityEvent event) {
```

Comment:
It can be but we'd have to draw a ray from the player for every single PlayerInteractEvent, which I don't think would be worth it given how there is still the right click defuse.